charts/kube-vip/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.4
+version: 0.5.0
```

Comment:
Why do you change the version without any other chart change?

Reply:
Hey @yaocw2020, because #22 got changed without a version bump and I was not aware that the chart-releaser-action actually expects that in the default mode.

Reply:
The errors lint-test is complaining about are unrelated to my change.